docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+timesketch:
+    build: .
+    ports:
+        - "5000:5000"
+    links:
+        - elasticsearch
+        - postgres
+    environment:
+        - POSTGRES_USER=timesketch
+        - POSTGRES_PASSWORD=password
+        - TIMESKETCH_USER=admin
+        - TIMESKETCH_PASSWORD=password
+        - POSTGRES_ADDRESS=postgres # Use hostname linking
+        - POSTGRES_PORT=5432
+        - ELASTIC_ADDRESS=elastic # Use hostname linking
+        - ELASTIC_PORT=9200
+    restart: always
+elasticsearch:
+    image: elasticsearch:2.4.1
+    command: elasticsearch -Des.node.name="TimesketchNode" -Des.script.groovy.sandbox.enabled=true
```

Comment:
Interesting. That does present a challenge. We could potentially mount the directory containing the scripts as a Docker volume for the Elastic container and then override the default Elastic command to run our scripts as well (or whatever needs to happen with them).